lighthouse-core/gather/gatherers/css-usage.js

```diff
@@ -44,11 +45,6 @@ class CSSUsage extends Gatherer {
       return driver.sendCommand('CSS.disable')
         .then(_ => driver.sendCommand('DOM.disable'))
         .then(_ => results.ruleUsage);
-    }).catch(err => {
```

Comment:
yeah, I thought of this immediately after opening the PR and it will fail :) need to add a nicer fix